python/UDF/requirements.txt

```diff
@@ -0,0 +1,5 @@
+boto3
+botocore
+pyarrow
```

Comment:
the example is only using boto3 and botocore. We should only include those libraries